package.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Do we need the D3.js dependency?

Reply:
I imagine the D3.js dependency will be introduced along with the carbon graphs package after the initial mono repo is setup.